modules/monitoring/files/usr/lib/nagios/plugins/check_reboots_required

```diff
@@ -21,12 +21,20 @@ def hostname_with_node_name(hostname, machines_names_hash)
   else
     hostname
   end
+end
 
+def print_hostnames(hostnames, machines_names_hash)
+  hostnames.sort.each do |hostname|
+    hydrated_name = hostname_with_node_name(hostname, machines_names_hash)
+    puts "- #{hydrated_name}"
+  end
 end
 
 all_machines = `/usr/local/bin/govuk_node_list`.split("\n")
 
-unsafe_machines = `/usr/local/bin/govuk_node_list --puppet-class govuk_safe_to_reboot::careful,govuk_safe_to_reboot::no`.split("\n")
+out_of_hours_unsafe_machines = `/usr/local/bin/govuk_node_list --puppet-class govuk_safe_to_reboot::no`.split("\n")
+in_hours_unsafe_machines = `/usr/local/bin/govuk_node_list --puppet-class govuk_safe_to_reboot::careful`.split("\n")
+unsafe_machines = out_of_hours_unsafe_machines + in_hours_unsafe_machines
```

Comment:
Minor: this seems to be used in a repetitive "+" everywhere. We could just define "machine_to_reboot" to DRY it up everywhere.

Reply:
Yep, I couldn't decide how far to refactor this script because to be honest it could do with completely re-writing... but anyway, I've fixed this in 71aaae77054faff900420d19b7f079bd686a0dab.